server/svix-server/src/v1/endpoints/endpoint.rs

```diff
@@ -691,4 +711,366 @@ mod tests {
         patched_hdrs.update_model(&mut model);
         assert_eq!(model.headers, Set(Some(updated_hdrs)));
     }
+
+    fn endpoint_in(url: &str) -> EndpointIn {
+        EndpointIn {
+            url: url.to_owned(),
+            version: 1,
+            ..Default::default()
+        }
+    }
+
+    async fn post_endpoint_default(
+        client: &TestClient,
+        app_id: &str,
+        ep_url: &str,
+    ) -> Result<EndpointOut> {
+        client
+            .post(
+                &format!("api/v1/app/{}/endpoint/", app_id),
+                endpoint_in(ep_url),
+                StatusCode::CREATED,
+            )
+            .await
+    }
+
+    async fn post_endpoint_in(
+        client: &TestClient,
+        app_id: &str,
+        ep: EndpointIn,
+    ) -> Result<EndpointOut> {
+        client
+            .post(
+                &format!("api/v1/app/{}/endpoint/", app_id),
+                ep,
+                StatusCode::CREATED,
+            )
+            .await
+    }
+
+    async fn get_endpoint(client: &TestClient, app_id: &str, ep_id: &str) -> Result<EndpointOut> {
+        client
+            .get(
+                &format!("api/v1/app/{}/endpoint/{}/", app_id, ep_id),
+                StatusCode::OK,
+            )
+            .await
+    }
+
+    async fn get_404(client: &TestClient, app_id: &str, ep_id: &str) -> Result<()> {
+        // Deserialize into a Value because it a basic JSON structure saying "Entity not found"
+        let _: serde_json::Value = client
```

Comment:
Here's the fix so that we can just use EmptyResponse: https://github.com/svix/svix-webhooks/pull/317